unstructured_inference/models/chipper.py

```diff
@@ -564,7 +663,7 @@ def element_overlap(
         ]
 
         for check_element in elements:
-            if check_element == element:
+            if check_element == element or not check_element.bbox:
```

Comment:
we have encountered cases where the bbox is not `None` but some of the attributes are e.g., bbox is like `(x1=1, y1=0, x2=None, y2=None)`. This condition wouldn't validate it. Maybe we should add a method to validate bbox to both check it is not None and do some check like https://github.com/Unstructured-IO/unstructured-inference/blob/main/unstructured_inference/inference/elements.py#L86-L88

Reply:
Done, created the function and updated where it was relevant.